platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py

```diff
@@ -316,6 +320,22 @@
                 THERMAL_DEV_FAN_AMBIENT
             ]
         )
+    },
```

Comment:
Why it was added last and not by the order of systems ? I see that you numbered them.

Reply:
It was added in the end because the order of is list should match the indexes stored in the array platform_dict. I didn't want to change the values in platform_dict for every switch, so I just added my switch as the last index.